protoarray/src/main/java/tech/pegasys/artemis/protoarray/ProtoArrayForkChoiceStrategy.java

```diff
@@ -278,11 +280,12 @@ public boolean containsBlock(Bytes32 blockRoot) {
    */
   static List<Long> computeDeltas(
       Map<Bytes32, Integer> indices,
-      ElasticList<VoteTracker> votes,
+      ReadOnlyStore store,
```

Comment:
This will need to be a mutable store because votes are being updated.

Reply:
Makes sense.